wagtail_storages/management/commands/fix_document_acls.py

```diff
@@ -1,6 +1,11 @@
 from django.core.management.base import BaseCommand, CommandError
 
-from wagtail.core.models import Collection
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION >= (3, 0):
```

Comment:
**suggestion** using a try accept block here might save one import of VERSION

Reply:
Resolved in https://github.com/torchbox/wagtail-storages/pull/28/commits/1ab46ab175ee5f01394aab82548a7d4a3f75624d